### PR TITLE
Update github-actions.md

### PR DESCRIPTION
**Pull request by akramsystems**
This step was missing from the optimize docker build using cache, without it the build fails with the following error message

```
Error: buildx failed with: ERROR: cache export feature is currently not supported for docker driver. Please switch to a different driver (eg. "docker buildx create --use")
```

### CODE REVIEW

File: ci-cd/github-actions.md [hunk 1]
```diff
@@ -148,14 +148,17 @@ step:
...
         uses: docker/login-action@v1
         with:
           username: ${{ secrets.DOCKER_HUB_USERNAME }}
           password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
-      -
-        name: Build and push
+      
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
```
Comment (crazy-max) [inline ci-cd/github-actions.md:159]:
```suggestion
        uses: docker/setup-buildx-action@v2
```

**Review by crazy-max — CHANGES_REQUESTED** [review]

**Comment by akramsystems** [comment]
@crazy-max updated sorry for the delay

**Review by akramsystems — COMMENTED** [review]
applied changes requested

File: ci-cd/github-actions.md [hunk 2]
```diff
@@ -148,13 +148,16 @@ step:
...
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
         with:
           username: ${{ secrets.DOCKER_HUB_USERNAME }}
           password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
-      -
+      
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
```
Comment (crazy-max) [inline ci-cd/github-actions.md:160]: * Why downgrading to `docker/login-action@v1`?
* `-` missing for `Build and push`
* Please keep the layout as it is and just add the `Set up Docker Buildx` step in your changes.

**Pull request closed without merging**